.github/workflows/commit.yml

```
@@ -0,0 +1,138 @@
+---
+name: commit
+
+on:
+  push:
+
+
+jobs:
+  test:
+    name: lint and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: build
+        run: make build
+      - name: Archive Catalog Docker image
+        run: docker save -o ckan/image-catalog.tar datagov/catalog.data.gov:latest
+      - name: Archive Solr Docker image
+        run: docker save -o ckan/image-solr.tar datagov/catalog.data.gov.solr:latest
+      - name: Archive DB Docker image
+        run: docker save -o ckan/image-db.tar datagov/catalog.data.gov.db:latest
+      - uses: actions/upload-artifact@v2
+        with:
+          name: docker-images
+          path: ckan/
```

Instead of archiving this directory, which has some code in it in addition to the tar archives, let's just artifact the individual files.
```suggestion
          path: |
            ckan/image-catalog.tar
            ckan/image-solr.tar
            ckan/image-solr.tar
```